maven/lib/dependabot/maven.rb

```diff
@@ -22,7 +22,7 @@
   register_display_name_builder(
     "maven",
     lambda { |name|
-      artifact_id = name.split(":").last
+      _, artifact_id, = name.split(":")
```

Comment:
I think something like this would be a bit clearer and shouldn't make rubocop complain
```suggestion
      _group_id, artifact_id, _classifier = name.split(":")
```